pkgs/development/python-modules/dbt/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchPypi, buildPythonPackage, autopep8 }:
+
+buildPythonPackage rec {
+  pname = "dbt";
+  version = "0.19.0";
+
```

Only [Python 3.7 and above](https://github.com/dbt-labs/dbt-core/blob/main/setup.py#L83) is supported.

```suggestion
  version = "0.19.0";

  disabled = pythonOlder "3.7";
```